packages/renderer/src/lib/kube/KubePlayYAML.svelte

```diff
@@ -174,24 +161,21 @@ async function playKubeFile(): Promise<void> {
   runStarted = false;
 }
 
-onMount(async () => {
-  providerUnsubscribe = providerInfos.subscribe(value => {
-    providers = value;
-  });
-});
-
-onDestroy(() => {
-  if (providerUnsubscribe) {
-    providerUnsubscribe();
-  }
-});
-
 function goBackToPodsPage(): void {
   // redirect to the pods page
   handleNavigation({
     page: NavigationPage.PODMAN_PODS,
   });
 }
+
+function toggle(choice: 'podman' | 'custom'): void {
+  userChoice = choice;
+
+  // reset custom content when switching away from custom
+  if (choice === 'podman') {
```

Comment:
question: do we want to keep this feature of clearing the yaml?

Reply:
this is out of scope of this PR, we could have an issue, but as this is just a migration we should not change any logic for the user :/